pkg/workspace/project/project.go

```diff
@@ -23,6 +23,7 @@ type Project struct {
 	ApiKey      string                     `json:"-"`
 	Target      string                     `json:"target" validate:"required"`
 	State       *ProjectState              `json:"state,omitempty" validate:"optional"`
+	Identity    string                     `json:"identity" validate:"required"`
```

Comment:
@the-johnwick no, it should be both in `Project` and `ProjectConfig`

Reply:
Because it needs to be stored in the `Project` and saved in the database as well. We need to know which project used which git provider.

When creating a project from a project config, the values from the project config are copied into the project.

Reply:
`ProjectConfig` and `Project` are 2 completely unrelated models.

The only connection to them is that properties from `ProjectConfig` are copied into `Project` during the creation process. Other than that, nothing connects them so both the models should include `GitProviderConfigId`